bridges/IPBBridge.php

```diff
@@ -68,7 +68,7 @@ public function collectData(){
 			case $this->isTopic($html):
 				$this->collectTopic($html, $limit);
 				break;
-			case $this->isForum($html);
+			case $this->isForum($html):
```

Comment:
Yes. The semicolon `;` is valid php syntax here and does what you expect. PSR2 requires the colon `:`.